### PR TITLE
create database

### PR DESCRIPTION
**Pull request by Eiritel**

### CODE REVIEW

File: database/create_connection.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+def create_db_connection():
+    db_connection = sqlite3.connect(STORAGE_CONFIG['DB_NAME'], check_same_thread = False)
+
+    questions_table.create_table(db_connection)
+    users_quiz_progress.create_table(db_connection)
+    users_table.create_table(db_connection)
+
+    return db_connection
```
Comment (AlexSkvorz) [inline database/create_connection.py:16]: Missing newline at the end of file

File: database/questions_table.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        cursor.execute('''
+            CREATE TABLE questions (
+            quiz_id INTEGER PRIMARY KEY,
+            topic TEXT,
+            question TEXT,
+            answers TEXT,
+            correct_answer TEXT)
+            ''')
+
+    db_connection.commit()
```
Comment (AlexSkvorz) [inline database/questions_table.py:19]: Missing newline at the end of file

File: database/questions_table.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+
+def create_table(db_connection):
+    cursor = db_connection.cursor()
+
+    cursor.execute("SELECT name FROM sqlite_master WHERE TYPE = 'table' AND NAME = 'questions_table';")
```
Comment (AlexSkvorz) [inline database/questions_table.py:6]: What table name should check in this method?

File: database/questions_table.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    query_result = cursor.fetchone()
+
+    if not query_result:
+        cursor.execute('''
+            CREATE TABLE questions (
```
Comment (AlexSkvorz) [inline database/questions_table.py:11]: !

File: database/users_quiz_progress.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+
+
+def create_table(db_connection):
+    cursor = db_connection.cursor()
+
+    cursor.execute("SELECT name FROM sqlite_master WHERE TYPE = 'table' AND NAME = 'users_quiz_progress';")
```
Comment (AlexSkvorz) [inline database/users_quiz_progress.py:6]: What table name should check in this method?

File: database/users_quiz_progress.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+            user_id INTEGER,
+            quiz_id INTEGER,
+            completed BOOL,
+            score INTEGER,
+            PRIMARY KEY (user_id, quiz_id),
+            FOREIGN KEY (user_id) REFERENCES Users(user_id) ON DELETE CASCADE,
+            FOREIGN KEY (quiz_id) REFERENCES Questions(quiz_id) ON DELETE CASCADE)
+            ''')
+
+    db_connection.commit()
```
Comment (AlexSkvorz) [inline database/users_quiz_progress.py:21]: Missing new line at the end of file

File: database/users_table.py [hunk 7]
```diff
@@ -0,0 +1,17 @@
+
+
+def create_table(db_connection):
+    cursor = db_connection.cursor()
+
+    cursor.execute("SELECT name FROM sqlite_master WHERE TYPE = 'table' AND NAME = 'users_table';")
```
Comment (AlexSkvorz) [inline database/users_table.py:6]: What table name should check in this method?

File: database/users_table.py [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+    if not query_result:
+        cursor.execute('''
+            CREATE TABLE users
+            (user_id INTEGER PRIMARY KEY,
+            username TEXT,
+            role INTEGER)
+            ''')
+
+    db_connection.commit()
```
Comment (AlexSkvorz) [inline database/users_table.py:17]: Missing newline at the end of file

File: main.py [hunk 9]
```diff
@@ -1,10 +1,11 @@
 import asyncio
-from telegram_bot.quiz_bot import QuizBot
 
+from telegram_bot.quiz_bot import QuizBot
+from database.create_connection import create_db_connection
```
Comment (AlexSkvorz) [inline main.py:5]: Missing spaces between imports and function

File: database/create_connection.py [hunk 10]
```diff
@@ -0,0 +1,16 @@
+import sqlite3
+
+from config.storage_config import STORAGE_CONFIG
+from database import questions_table
+from database import users_quiz_progress
+from database import users_table
+
+
+def create_db_connection():
+    db_connection = sqlite3.connect(STORAGE_CONFIG['DB_NAME'], check_same_thread = False)
```
Comment (AlexSkvorz) [inline database/create_connection.py:10]: Why are there spaces between check_same_thread and False?

File: database/create_connection.py (hunk 1 above)
Comment (Eiritel) [reply database/create_connection.py:16]: pizdets

**Pull request merged**